documents/Specification/MaterialX.PBRSpec.md

```diff
@@ -243,6 +243,21 @@ The PBS nodes also make use of the following standard MaterialX types:
     * `normal` (vector3): Normal vector of the surface. Defaults to world space normal.
     * `mode` (uniform string): Selects between `conty_kulla` and `zeltner` sheen models. Defaults to `conty_kulla`.
 
+<a id="node-hair-chiang-bsdf"> </a>
+
+* **`hair_chiang_bsdf`**: Constructs a hair BSDF based on the Chiang hair shading model[^Chiang2016]. This node does not support vertical layering.
```

Comment:
For consistency with existing MaterialX PBR nodes such as `burley_diffuse_bsdf`, perhaps the name of this new node should be `chiang_hair_bsdf`?

Reply:
I was wondering about that too, though other nodes related to hair are called `mx_hair_*` then I took the other consistency. Either works for me so I'm fine to follow your decision.

Reply:
If you're open to this convention, I'd definitely recommend `chiang_hair_bsdf` for consistency with other PBR nodes, and the helper functions starting with `mx_hair_chiang_*` could be updated to `mx_chiang_hair_*` to match.

Reply:
Sounds good. Technically, the method of `mx_hair_absorption_from_melanin` isn't from Chiang et al., so maybe we should rename those to:

- `mx_chiang_hair_bsdf`
- `mx_chiang_hair_roughness`
- `mx_chiang_hair_absorption_from_color`
- `mx_deon_hair_absorption_from_melanin`

How do they sound for you?

Reply:
Those names look good to me!

Reply:
Great, the changes submitted.